pkgs/top-level/python-packages.nix

```diff
@@ -1255,7 +1255,7 @@ in {
 
   cnvkit = callPackage ../development/python-modules/cnvkit { };
 
-  coapthon3 = callPackage ../development/python-modules/coapthon3 { };
+  coapthon3 = disabledIf isPy27 (callPackage ../development/python-modules/coapthon3 { });
```

Comment:
Please do this in the package itself, e.g.:

```nix
  disabled = isPy27;
```

conventionally directly under the `version =`